src/app/clients/getClientsData.ts

```diff
@@ -33,11 +34,18 @@ const getClientsData = async (
 
     query = query.range(startIndex, endIndex);
 
+    query = query.abortSignal(abortSignal);
+
     const { data: clients, error: clientError } = await query;
 
     if (clientError) {
-        const logId = await logErrorReturnLogId("Error with fetch: Clients", clientError);
-        throw new DatabaseError("fetch", "clients", logId);
+        let logId;
+        if (abortSignal.aborted) {
+            logId = await logInfoReturnLogId("Aborted fetch: client table count", clientError);
+        } else {
+            logId = await logErrorReturnLogId("Error with fetch: client table count", clientError);
+        }
```

Comment:
```suggestion
        const logId = abortSignal.aborted
            ? await logInfoReturnLogId("Aborted fetch: client table count", clientError);
            : await logErrorReturnLogId("Error with fetch: client table count", clientError);
```

(same change could be made in getClientsCount and fetchParcelTableData)

Reply:
nice